tribe-ext-organizer-notifications.php

```diff
@@ -42,8 +42,26 @@ public function init() {
 
 			// EDD
 			add_action( 'event_ticket_edd_attendee_created', [ $this, 'generate_email' ], 10, 2 );
+
+			// Tickets Commerce
+			add_action( 'tec_tickets_commerce_flag_action_generated_attendees', [ $this, 'generate_email_from_ticket' ], 10, 7 );			
+		}
+
+		/**
+		 * Generate organizer email from ticket
+		 *
+		 * @param $other
+		 * @param $ticket
+		 */
+		public function generate_email_from_ticket( $attendees, $ticket, $order, $new_status, $old_status ) {
+
+			// Get the Event ID the ticket is for
+			$event_id = $ticket->get_event_id();
+
+			$this->generate_email( $event_id );
```

Comment:
The generate_email method collects the event id as the second parameter. This was only done because all the original hooks had event ID as the second, but some other data as the first. Looking at this years later, this is a bad way to handle things.